src/index.php

```diff
@@ -1,5 +1,7 @@
 <?php
```

Comment:
You lower-cased the file name. Wanna make sure you avoid that for case-sensitive file systems. Also, convention is that class filenames are capitalized. :)